server/app/app.go

```diff
@@ -84,7 +84,7 @@ func (a *App) loadConfigurationDefaults() {
 	a.config.SetDefault("kafka.producer.idempotent", false)
 	a.config.SetDefault("kafka.producer.net.keepAlive", "60s")
 	a.config.SetDefault("kafka.producer.brokers", "localhost:9192")
-	a.config.SetDefault("kafka.producer.maxMessageBytes", 1000000)
+	a.config.SetDefault("kafka.producer.maxMessageBytes", 5242880)
```

Comment:
Let's keep the default at 1MB, it's Kafka's default as well (we can change this individually per deployment depending on the Kafka cluster's setting)